pages/infrastructure-as-code-library/_modal.html

```diff
@@ -21,3 +21,26 @@ <h4 class="h2 modal-title margin-top-none" id="modalPublicRepoTitle">You're abou
     </div>
   </div>
 </div>
+
+<div class="modal fade" id="modalPrivateRepo" tabindex="-1" role="dialog" aria-labelledby="modalPrivateRepoTitle">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header text-center">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><i class="fa fa-fw fa-close"></i></button>
+      </div>
+      <div class="modal-body text-center">
+        <h4 class="h2 modal-title margin-top-none" id="modalPrivateRepoTitle">You're about to view a private repo.</h4>
+        <p>
+          Private repos are available to <a href="/pricing">Gruntwork Library subscribers</a>.
```

Comment:
s/Gruntwork Library/Infrastructure as Code Library/

Also, you could add `#infrastructure-as-code-library` to the link.